bin/cf_deploy.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+set -o errexit
+set -o pipefail
+set -x
+
+cloud_gov=${CF_API:-https://api.fr.cloud.gov}
+
+app=${1}
+org=${2}
+branch=${3}
+
+# Get the space that corresponds with the branch name
+if [[ ${branch} == "develop" ]]; then
+    echo "Branch is develop, deploying to dev space"
+    space="dev"
+elif [[ ${branch} == release* ]]; then
+    echo "Branch starts with release, deploying to stage space"
+    space="stage"
+elif [[ ${branch} == "master" ]]; then
+    echo "Branch is develop, deploying to prod space"
```

Comment:
Shouldn't this comment say "Branch is master..."

Reply:
Fixed, thanks!